gutenberg/blocks/blocks.php

```diff
@@ -646,6 +646,17 @@ private function parseDateFilter($dateString) {
       return $result;
     }
 
+    /**
```

Comment:
What is the purpose of this change?

Reply:
This is used to modify shown options from YSO. F.ex "Tekniikka (erikoisalat)" -> "Tekniikka" to make options clearer.

Reply:
This sounds like a problem with data and thus should not be fixed in code. Correct me if i'm wrong

Reply:
This feature was requested. If it can be accepted to be a part of this plugin, great. But if it's not acceptable, I will delete this part from the PR.

Reply:
Could this be moved into options, because most of users probably do not need such a feature?